lib/beaker/host_prebuilt_steps.rb

```diff
@@ -13,7 +13,7 @@ module HostPrebuiltSteps
     SLEEPWAIT = 5
     TRIES = 5
     UNIX_PACKAGES = ['curl', 'ntpdate']
-    FREEBSD_PACKAGES = ['curl', 'perl5']
+    FREEBSD_PACKAGES = ['curl', 'perl5|perl']
```

Comment:
I think that's ok, running FreeBSD in a docker container is a pretty uncommon edgecase